db/order_products.js

```diff
@@ -31,6 +31,55 @@ const addProductToOrder = async (order_product) => {
     }
 }
 
+const getOrderProductById = async (id) => {
+    try {
+        const { rows: [ order_products ] } = await client.query (`
+            SELECT *
+            FROM order_products
+            WHERE id = $1;
+        `, [id])
+
+        return order_products;
+    } catch (error) {
+        throw error
+    }
+}
+
+const updateOrderProduct = async ({id, price, quantity}) => {
+    
+    try {                 
+            const { rows: [ order_product ] } = await client.query(`
+                UPDATE order_products
+                SET price = $1, quantity = $2
```

Comment:
`price` and `quantity` will always be updated, so maybe just update the one that we want? (like if you're in the cart and you just want to edit the quantity of an item, then how will we do that?)

Reply:
Fixed